src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -149,331 +93,87 @@ export default function CDDetails() {
-                                >
-                                    <TriggerOutput
-                                        fullScreenView={fullScreenView}
-                                        syncState={syncState}
-                                        triggerHistory={triggerHistory}
-                                        setShowTemplate={setShowTemplate}
-                                        setDeploymentHistoryList={setDeploymentHistoryList}
-                                        deploymentHistoryList={deploymentHistoryList}
-                                        deploymentAppType={deploymentAppType}
-                                        isBlobStorageConfigured={blobStorageConfiguration?.result?.enabled || false}
-                                    />
-                                </Route>
-                            )}
-
-                            {!!envId && triggerHistory?.size === 0 && (
-                                <NoCDTriggersView environmentName={environment?.environmentName} />
-                            )}
-                            {pathname.includes('/logs') && (
-                                <Tippy
-                                    placement="top"
-                                    arrow={false}
-                                    className="default-tt"
-                                    content={fullScreenView ? 'Exit fullscreen (f)' : 'Enter fullscreen (f)'}
-                                >
-                                    {fullScreenView ? (
-                                        <ZoomOut
-                                            className="zoom zoom--out pointer"
-                                            onClick={(e) => setFullScreenView(false)}
-                                        />
-                                    ) : (
-                                        <ZoomIn
-                                            className="zoom zoom--in pointer"
-                                            onClick={(e) => setFullScreenView(true)}
-                                        />
-                                    )}
-                                </Tippy>
-                            )}
-                        </div>
-                    </>
-                )}
-                <Switch>
-                    <Route
-                        path={`${path}${URLS.DEPLOYMENT_HISTORY_CONFIGURATIONS}/:historyComponent/:baseConfigurationId(\\d+)/:historyComponentName?`}
-                        render={(props) => (
-                            <DeploymentHistoryDetailedView
-                                showTemplate={showTemplate}
-                                setShowTemplate={setShowTemplate}
+            <div className={`ci-details  ${fullScreenView ? 'ci-details--full-screen' : ''}`}>
+                <div className="ci-details__history">
+                    {!fullScreenView && (
+                        <Sidebar
```

Comment:
Do we have to render `ci-details__history` if not in full-screen view?

Reply:
Yes